ui/src/components/song/SongCard.tsx

```diff
@@ -78,35 +99,41 @@ const SongCard = (props: SongCardProps) => {
           {filterData?.display?.lyricsPreview === true ||
           filterData?.display?.lyricsPreview === undefined ? (
             <>
```

Comment:
<img width="589" alt="Screenshot 2023-12-17 at 9 34 13 AM" src="https://github.com/hmcc-global/psalted-2/assets/26876443/97f1a272-5f6b-4578-8381-0d11f7ebd5d9">

line 84 have this warning error. so you can remove the <></> at the very top